sources/packages/forms/src/form-definitions/supportingusersdashboard.json

```diff
@@ -1,12 +1,17 @@
 {
+  "title": "Supporting Users Dashboard",
+  "display": "form",
   "type": "form",
+  "name": "supportingusersdashboard",
+  "path": "supportingusersdashboard",
   "tags": [
     "common"
   ],
-  "owner": "andrew.signori@gov.bc.ca",
   "components": [
     {
       "label": "HTML",
+      "labelWidth": "",
+      "labelMargin": "",
       "tag": "p",
       "className": "",
       "attrs": [
```

Comment:
Please add the customClass category-header-x-large for the welcome content to sync with figma.

Reply:
ok. Added. Thanks!